### ISSUE TITLE
Volumes for {assessment,course}Config?

### ISSUE THREAD

**Issue by edmcman** [autolab/autolab-docker#1]
Shouldn't there be volumes for assessmentConfig and courseConfig too?

**Comment by jez** [comment]
Yes, this is an issue with the current setup. There also needs to be a volume for the `gradebooks` folder. We haven't settled on a way to handle this due to the way these folders are automatically created and some subtleties with symlinks involved in the deployment at CMU (which existed before the Docker setup was an option).

**Comment by edmcman** [comment]
Might I suggest at least adding a note to the instructions on setting up a production server in the meanwhile?

**Comment by edmcman** [comment]
This is still broken...

**Comment by edmcman** [comment]
attachments should be a volume too
